packages/nextjs-kit/src/components/footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Link from 'next/link';
+import { FooterMenuItem } from '../types/index';
+
+interface FooterMenuProps {
+  footerMenuItems: FooterMenuItem[];
+  children: JSX.Element;
+}
+
+/**
+ * This is a Footer component.
+ *
+ * @param {Props} footerMenuItems holds the linkText, href, and parent of each route to be displayed
+ * @param {Props} children  copyright display and information to be passed in by the user
+ * @returns {React.JSX.Element}
+ */
```

Comment:
I think there's actually just the one param (called `props`) which in this case is of type `FooterMenuProps`
In this way we need to call out the first param, then the nested ones. See https://typedoc.org/tags/param/
```suggestion
/**
 * This is a Footer component.
 *
 * @param {FooterMenuProps} props - The props needed for the footer component
 * @param {FooterMenuItem[]} props.footerMenuItems - An array of footer menu items
 * @example
 * ```
 * const footerMenuItems = [
 *  {
 *   href: '/',
 *   title: 'Home',
 *  },
 * {
 *  href: '/articles',
 *  title: 'Articles',
 *  parent: 'home',
 * },
 * ...
 *  ]
 * ```
 * @param {ReactChildren} props.children
 * @returns {JSX.Element} A footer component with a nav menu
 */
```